candle-pyo3/src/lib.rs

```diff
@@ -330,6 +334,13 @@ impl PyTensor {
         PyTuple::new(py, self.0.dims()).to_object(py)
     }
 
+    #[getter]
+    /// Gets the tensor's element count.
+    /// &RETURNS&: int
+    fn nelements(&self) -> usize {
```

Comment:
The pytorch version has `nelement` and `numel` for this. Maybe we could use these name instead of `nelements`?

Reply:
Gonna rename it to `nelement` 👍